chevah_build

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bash
+#
+# Chevah Build Script for CPython distribution.
+#
+
+PYTHON_UPSTREAM_VERSION='2.7.8'
+PYTHON_BUILD_VERSION='.c1'
+LIBFFI_UPSTREAM_VERSION='3.2.1'
+
+# Import shared code.
+. ./functions.sh
+
+# List of OS packages required for building Python.
+UBUNTU_PACKAGES="gcc make libssl-dev zlib1g-dev m4 texinfo"
+RHEL_PACKAGES="gcc make openssl-devel zlib-devel m4 texinfo"
+SLES_PACKAGES="gcc make libopenssl-devel zlib-devel m4 texinfo"
```

Comment:
we  will no longer build readline/sqlite/pysqlite from this repo so we don't need to install the ncurses lib here.